cli/src/commands/fix.rs

```diff
@@ -125,6 +126,11 @@ pub(crate) struct FixArgs {
     /// Fix only these paths
     #[arg(value_hint = clap::ValueHint::AnyPath)]
     paths: Vec<String>,
+    /// Fix unchanged files in addition to changed ones. If no files are
+    /// specified, all files in the repo will be fixed.
+    // TODO: use -L for --all-lines, and avoid -A
+    #[arg(long, short = 'F')]
+    all_files: bool,
```

Comment:
nit: maybe `--include-unchanged-files` or something is better?

`fix --all-files PATH` looks a bit odd.